Main/src/Coffee/Action.java

```diff
@@ -0,0 +1,186 @@
+            Action.dcups -= cup;
+            System.out.println("Price is 6$ for cup. Thank you for your purchase, please take your cappuccino.");
+        }
+    }
+    public static boolean checkEspressoAvailability (int cup){
+        float avOfWater=Action.water;
+        float avOvCoffeeBeans = Action.coffeeBeans;
+        float avOfdCups = Action.coffeeBeans;
+        System.out.println("Checking for Espresso availability...");
+        avOfWater = avOfWater/(250*cup);
+        avOvCoffeeBeans = avOvCoffeeBeans/(16*cup);
+        avOfdCups = avOfdCups/cup;
+        if (avOfWater>=1 & avOvCoffeeBeans>=1 & avOfdCups>=1){                // нужно округление
+            System.out.println("You can order Espresso!");
+            System.out.println("Available cups " + FindMin.min(avOfWater, avOvCoffeeBeans, avOfdCups));
+            return true;
+        }
+        else {System.out.println("No enough ingredients, Please fill Machine");
+        return false;                                                                                                       //вернуться в степ1
+        }
+    }
+
+    public static boolean checkLatteAvailability (int cup){
+        float avOfWater=Action.water;
+        float avOvCoffeeBeans = Action.coffeeBeans;
+        float avmilk = Action.milk;
+        int avOfdCups = Action.coffeeBeans;
+        System.out.println("Checking for Latte availability...");
+        avOfWater = avOfWater/(350*cup);
+        avmilk = avmilk/(75*cup);
+        avOvCoffeeBeans = avOvCoffeeBeans/(20*cup);
+        avOfdCups = avOfdCups/cup;
+        if (avOfWater>=1 & avmilk>=1 & avOvCoffeeBeans>=1 & avOfdCups>=1){
+            System.out.println("You can order Latte!");
+            System.out.println("Available cups " + FindMin.min(avOfWater, avmilk, avOvCoffeeBeans, avOfdCups));
+            return true;
+            }
+        else {
+            System.out.println("No enough ingredients, Please fill Machine");
+            return false;                                                                                                   //вернуться в степ1
+        }
+    }
+    public static boolean checkCappuccinoAvailability (int cup){
+        float avOfWater=Action.water;
+        float avOvCoffeeBeans = Action.coffeeBeans;
+        float avmilk = Action.milk;
+        int avOfdCups = Action.coffeeBeans;
+        System.out.println("Checking for Cappuccino availability...");
+        avOfWater = avOfWater/(200*cup);
+        avmilk = avmilk/(100*cup);
+        avOvCoffeeBeans = avOvCoffeeBeans/(12*cup);
+        avOfdCups = avOfdCups/cup;
+        if (avOfWater>=1 & avmilk>=1 & avOvCoffeeBeans>=1 & avOfdCups>=1){
+            System.out.println("You can order Cappuccino!");
+            System.out.println("Available cups " + FindMin.min(avOfWater, avmilk, avOvCoffeeBeans, avOfdCups));
+            return true;
+        }
+        else {System.out.println("No enough ingredients, Please fill Machine");
```

Comment:
@GayaneGulanyan jan please format overall flow style is important , you can use code formatters for that . Research some plugin for code formating for inteliji

Reply:
done